### PR TITLE
Use xdg-open to open URL

### PR DESCRIPTION
**Pull request by gitressa**
This line is formatted like the other lines, as a command, but I guess "open" means copy-pasting the URL into a browser:

```
# Visit the local site
open https://hello-lando.lndo.site
```

Maybe it's better use a command?

### CODE REVIEW

**Comment by xurizaemon** [comment]
`open` is an equivalent command to `xdg-open` found on another OS

`open` is the OS X version, and `xdg-open` is what you'll find on Linux. Not sure if there will be similar tools to consider on other OSes (eg Windows).

- https://ss64.com/bash/xdg-open.html
- https://ss64.com/osx/open.html

**Comment by gitressa** [comment]
Thanks for clarifying @xurizaemon. Should we do it like this?

```
# Visit the local site
# macOS
open https://hello-lando.lndo.site
# Linux
xdg-open https://hello-lando.lndo.site
```

File: getting-started/first-app.md [hunk 1]
```diff
@@ -67,7 +67,10 @@ lando composer require drush/drush
 lando drush site:install --db-url=mysql://drupal9:drupal9@database/drupal9 -y
 
 # Check out your new site!
-open https://hello-drupal9.lndo.site
+xdg-open https://hello-drupal9.lndo.site
```
Comment (xurizaemon) [inline getting-started/first-app.md:70]:
```suggestion
# Open https://hello-drupal9.lndo.site in your browser
```

**Review by xurizaemon — COMMENTED** [review]
I would say that the use of `open` or `xdg-open` complicates rather than simplifying this documentation, and it's better to instruct the user to open the URL than to supply a CLI command which may do the task.

I think that making the documentation "branch" for various OSes to achieve this would reduce clarity unnecessarily.

So, I think this PR should replace the use of a command with an instruction to open the URL in a browser, which ought to work for all cases.

My code suggestions in this review are two lines, but I think it should be a single line comment, I just couldn't work out how to get Github to let me "suggest" that using the web UI 😁

**Comment by xurizaemon** [comment]
You put it succinctly in the issue description!

> I guess "open" means copy-pasting the URL into a browser:

**Comment by gitressa** [comment]
Yes, let's do it like that, keeping it simple!

**Review by xurizaemon — APPROVED** [review]
I approve these changes!

**Pull request merged**